db.go

```diff
@@ -261,6 +263,50 @@ func (s *ColumnStore) DB(name string) (*DB, error) {
 
 	db.txPool = NewTxPool(db.highWatermark)
 
+	ctx := context.TODO()
```

Comment:
It doesn't seem unreasonable to me to add the context to the `DB` function parameters.

Reply:
Done